README.md

```diff
@@ -32,6 +32,5 @@ curl -L https://deno.land/x/install/install.py | python - v0.2.0
 
 **Install with PowerShell:**
 
-```
-iwr https://deno.land/x/install/install.ps1 -out install.ps1; .\install.ps1 v0.2.0
-```
+```powershell
+$install = iwr https://deno.land/x/install/install.ps1; iex "$install v0.2.0"
```

Comment:
This doesn't work, unfortunately.

```
PS C:\> $install = iwr https://raw.githubusercontent.com/artemmnad/deno_install/feature/powershell-specify-version/
install.ps1
PS C:\Users\wwwma> iex "$install v0.2.0"
Downloading https://api.github.com/repos/denoland/deno/releases/latest... done.
Determining latest Deno release: v0.2.5...  done.
Downloading https://github.com/denoland/deno/releases/download/v0.2.5/deno_win_x64.zip... done.
Extracting C:\Users\<redacted>\.deno\bin\deno_win_x64.zip into C:\Users\<redacted>\.deno\bin... done.
Removing C:\Users\<redacted>\.deno\bin\deno_win_x64.zip... done.

Deno was installed successfully.
Run deno --help to get started.

iex : The term 'v0.2.0' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ iex "$install v0.2.0"
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (v0.2.0:String) [Invoke-Expression], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShell.Commands.InvokeExpressionCommand
```

Reply:
How about this?

```powershell
icm ([scriptblock]::Create("$(iwr $url)")) -Args v0.2.0
```

Reply:
Looks great but you're right. 
It is significantly less readable and I think that readability is more important than not creating a temp file.